ddtrace/settings/config.py

```diff
@@ -62,6 +62,15 @@ def __init__(self):
         self.version = os.getenv("DD_VERSION") or self.tags.get("version")
         self.http_server.error_statuses = "500-599"
 
+        service_mapping = get_env("service", "mapping")
+        if service_mapping is not None:
+            try:
+                self.service_mapping = dict([(_.strip() for _ in m.split(":")) for m in service_mapping.split(",")])
```

Comment:
We can probably use this helper: https://github.com/DataDog/dd-trace-py/blob/8abf07fe06e7e834f26f40246ada828f8be1ab3a/ddtrace/utils/formats.py#L87-L116

Not sure if the error message makes as much sense, but the format is the same so might be fine? (or we can catch and re-throw with a better message?)

Reply:
Ah! Thanks, I should have looked around for something like this! Presumably, this is also used somewhere else, so it makes sense to have a homogeneous way of handling strings with similar format.